pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -211,6 +218,20 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 	return output.LaunchTemplate, nil
 }
 
+func (p *LaunchTemplateProvider) onCacheEvicted(key string, lt interface{}) {
+	p.Lock()
+	defer p.Unlock()
+	if _, expiration, _ := p.cache.GetWithExpiration(key); expiration.After(time.Now()) {
+		return
+	}
+	launchTemplate := lt.(*ec2.LaunchTemplate)
+	if _, err := p.ec2api.DeleteLaunchTemplate(&ec2.DeleteLaunchTemplateInput{LaunchTemplateId: launchTemplate.LaunchTemplateId}); err != nil {
+		p.logger.Debugf("unable to delete launch template, %v", err)
```

Comment:
Should this be an error?

Reply:
All of our other log lines start with a capital letter.

Reply:
I went back-and-forth on making this Error or not. I'm fine making it an Error but the reason I didn't is because this is not strictly required to operate correctly and we are adding a permission. But it's probably better to alert users sooner to update permissions if they miss it in the release notes.  I'm convinced again on making it error after writing this :) Updated and updated.